apps/vmq_bridge/priv/vmq_bridge.schema

```diff
@@ -88,6 +118,36 @@
                                                        hidden
                                                       ]}.
 
+%% @doc Configurea segment size (in Bytes) for the Bridge Queue.
+{mapping, "vmq_bridge.tcp.$name.segment_size", "vmq_bridge.config", [
+                                                             {default, 4096},
+                                                             {datatype, integer},
+                                                             {include_default, "br0"},
```

Comment:
bytesize?

Reply:
I think I should have been more precise, my suggestion was to use the datatype bytesize instead of integer, to make it clear what is expected. Apologies for the confusion.

Reply:
Done.